tests/attention/test_trtllm_gen_attention.py

```
@@ -1837,3 +1837,155 @@ def test_trtllm_batch_decode_spec(
+
+    # Reference: naive attention in float32
+    output_naive = naive_ragged_attention(
+        q, k_cache, v_cache, qo_indptr, kv_indptr, scale, causal
+    )
+
+    # TRT-LLM gen
+    output = torch.empty_like(output_naive)
+
+    skip_softmax_threshold_scale_factor = 1e-30 if skips_softmax else None
+
+    output_trtllm, lse_trtllm = flashinfer.prefill.trtllm_ragged_attention_deepseek(
```

Ruff (0.15.9)

[warning] Unpacked variable `lse_trtllm` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)